### DISCUSSION TITLE
Support `pointer` media query

### DISCUSSION THREAD

**Discussion by withinboredom** [tailwindlabs/tailwindcss#11869] [Ideas]
When building a touch-oriented interface (no matter the size), you often want more padding in buttons and other interactions. However, it doesn't seem that Tailwind supports the [pointer](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) media query that would allow creating better designs, instead of depending on the size of the screen (such as phones/tablets in portrait mode).

Context: https://news.ycombinator.com/item?id=37208969

**Comment by iohansson** [comment]
@withinboredom you can configure it. Check this https://tailwindcss.com/docs/plugins#adding-variants out

**Comment by elieven** [comment]
Any good reason this is not included by default? Seems an easy addition?
